### PR TITLE
toyokeizai.net  

### PR DESCRIPTION
**Pull request by SKEIDs**
moved from #103153

### CODE REVIEW

File: AnnoyancesFilter/sections/widgets.txt [hunk 1]
```diff
@@ -23,6 +23,7 @@ blog.livedoor.jp,duellinks.doorblog.jp,dragonquestwalk.blog.jp##.rss
...
 !
 !
+toyokeizai.net##.SURVIN_GUIDE_ELEMENT_SHOW_vrzubc.SURVIN_GUIDE_ELEMENT_SHOW_AFTER_1pab53s
```
Comment (zloyden) [inline AnnoyancesFilter/sections/widgets.txt:26]:
```
toyokeizai.net##div[class^="SURVIN_GUIDE_"]
```
Not `widgets` -> `popups`.

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -2299,8 +2299,12 @@ openloadpro.com,javfan.one,javtheater.com,jav.direct,javmix.tv##.__isboostOverCo
 9tsu.*,jshow.tv##.ads-system
 9tsu.*,jshow.tv##.cactus-sidebar-content > .widget_text
 ||rakuten.co.jp/com/img/thumb/logout/bnr/
+toyokeizai.net###adexchange
+toyokeizai.net###induction-banner
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:2303]: Please move it to the `AdGuard Annoyances - self-promo`.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/widgets.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/sections/widgets.txt]: ok.done

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt]: ok.done

**Pull request merged**